components/global.scss

```diff
@@ -67,6 +68,50 @@ $sides: (top: t, right: r, bottom: b, left: l);
   }
 }
 
+// UNITS: PIXELS!!
+$startPX: 0 !default;
+$endPX:  25 !default;
+
+// margin & padding m-1px through m-25px, p-1px through p-25px
+@each $property, $prop in $properties {
+  @for $i from $startPX through $endPX {
```

Comment:
Seems like you're still dealing with some inconsistent indention. Appears to be two spaces here, and tabs on lines 85 and following. Have you looked at https://editorconfig.org/?

Reply:
I use prettier but haven't set up a .prettierrc file on Ubuntu yet. Formatting command doesn't change anything so I probably need a config file first.